src/EmptyHandler/NoEmpty.php

```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Validator\SkipOnEmptyCallback;
+namespace Yiisoft\Validator\EmptyHandler;
 
-final class SkipNone
+final class NoEmpty
```

Comment:
```suggestion
final class NoneCriteria
```

Reply:
Or `Nothing` criteria.

Reply:
Current name is alright IMO. Alternatively it could be `Never`.